README.md

```diff
@@ -0,0 +1,11 @@
+# README
+
+# create databases
```

Comment:
👍 for adding readme with instructions. `#` is used only for title. Each section should use `##` instead (`###` for subsections and so on).